### ISSUE TITLE
Manga-Colorization-FJ中generator.pt文件无法下载

### ISSUE THREAD

**Issue by noidea-tony** [xiaogdgenuine/Doll#30]
@xiaogdgenuine 在Manga-Colorization-FJ仓库无法发Issues，只有在此仓库发起。https://github.com/xiaogdgenuine/Manga-Colorization-FJ 中提到的https://cubeatic.com/index.php/s/PcB4WgBnHXEKJrE/download 文件不存在，不知道是否可以重新给一个有效的地址呢
![image](https://user-images.githubusercontent.com/55474635/231975554-20bd4921-93a6-4d52-b9d5-4f007a9d0c41.png)

**Comment by jkjkmala** [comment]
Yes the file is missing .. can it be updated with a new link @xiaogdgenuine

**Comment by ADKYUSHU12321** [comment]
作者好像不更新看，哎

**Comment by tianya12612** [comment]
作者好像不更新了，哎
